front-end/packages/robustar/src/views/ImageList.vue

```diff
@@ -282,11 +279,8 @@ export default {
       maxPage: 0,
       imageList: [],
       splitLength: 1000,
-      classNames: [],
+      classNames: ['none'],
```

Comment:
Maybe we don't need a dummy value here. If a user has not selected a class, we can set `imageClass` to be `""`. When looking for start index, we can first check if it's empty. If it is, we simply start from 0. IDK what your opinion is but I personally think it's more elegant.

Reply:
If there is no placeholder like `none` here, users may find no way to reset image class back to empty, I think.